src/components/Comments/Comments.jsx

```diff
@@ -52,6 +52,19 @@ const Comments = () => {
         
     };
 
+    const handleDelete = (commentId, user) => {
+
+        if(selectedUser === user){
+        newsApi.delete(`comments/${commentId}`)
+        .then(() => {
+            confirm('Comment deleted');
+        })
+        .catch((error) => {console.log(error)})
+    } else {
+        alert('You cannot delete another users comment')
```

Comment:
i would argue for UI/UX purposes, other users shouldnt be able to see a delete button for anybody elses comments if they arent supposed to delete them?